src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -307,5 +318,35 @@ private static bool SupportsCSharpCodeActions(JToken token)
 
             return providesCodeActions && resolvesCodeActions;
         }
+
+        public override Task<JObject[]> UpdateConfigurationAsync(
+            OmniSharp.Extensions.LanguageServer.Protocol.Models.ConfigurationParams configParams,
```

Comment:
I'd add a using statement to alias this

```C#
using OmnisharpConfigurationParams = OmniSharp.Extensions.LanguageServer.Protocol.Models.ConfigurationParams;
```

Should also null check this parameter